### PR TITLE
#857 Added locale pt-br

### LINKED ISSUE #857: Add locale pt-br. I can help with translation.
Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your
needs please complete the below template to ensure we have the details to help. Thanks!

**Please check out the [documentation](https://pnp.github.io/sp-dev-fx-controls-react/) to see if your question is already addressed there. This will help us ensure our documentation is up to date.**

#### Category
[X ] Enhancement

[ ] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [  2.0.0   ]

_If you are not using the latest release, please update and see if the issue is resolved before submitting an issue._

#### Expected / Desired Behavior / Question
_If you are reporting an issue please describe the expected behavior. If you are suggesting an enhancement please
describe thoroughly the enhancement, how it can be achieved, and expected benefit. If you are asking a question, ask away!_

#### Observed Behavior
_If you are reporting an issue please describe the behavior you expected to occur when performing the action. If you are making a
suggestion or asking a question delete this section._

#### Steps to Reproduce
_If you are reporting an issue please describe the steps to reproduce the bug in sufficient detail to allow testing. If you are making
a suggestion or asking a question delete this section._

#### Submission Guidelines
_Delete this section after reading_
- All suggestions, questions and issues are welcome, please let us know what's on your mind.
- Remember to include sufficient details and context.
- If you have multiple suggestions, questions, or bugs please submit them in separate issues so we can track resolution.

Thanks!

### PR DESCRIPTION
**Pull request by kunj-sangani**
| Q               | A
| --------------- | ---
| Bug fix?        | [ X]
| Related issues?  | fixes #857 

#### What's in this Pull Request?

Please describe the changes in this PR. Sample description or details around bugs which are being fixed.

This will add locale for pt-br 
Added a file named pt-br.ts inside loc folder

### CODE REVIEW

**Comment by kunj-sangani** [comment]
@joelfmrodrigues 
I hope this helps :)

File: src/loc/pt-br.ts [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    StandardColorLightblue: "Azul claro",
+    StandardColorBlue: "Azul",
+    StandardColorDarkblue: "Azul escuro",
+    StandardColorPurple: "Roxo",
+
+    DatePickerMonthLongJanuary: "Janeiro",
+    DatePickerMonthShortJanuary: "Jan",
+    DatePickerMonthLongFebruary: "Fevereiro",
+    DatePickerMonthShortFebruary: "Fev",
+    DatePickerMonthLongMarch: "Marcha",
```
Comment (aaclage) [inline src/loc/pt-br.ts:173]: DatePickerMonthLongMarch: "Marcha", should be DatePickerMonthLongMarch: "Março",

File: src/loc/pt-br.ts [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerMonthShortMarch: "Março",
```
Comment (aaclage) [inline src/loc/pt-br.ts:174]: DatePickerMonthShortMarch: "Março", should be DatePickerMonthShortMarch: "Mar", the same for the other short names

Comment (kunj-sangani) [reply src/loc/pt-br.ts:174]: Hi @aaclage ,

Sorry will update all the changes and create a PR again 
One small question DatePickerMonthShortMay should be "Maio" right ?

Comment (aaclage) [reply src/loc/pt-br.ts:174]: > Hi @aaclage ,
> 
> Sorry will update all the changes and create a PR again
> One small question DatePickerMonthShortMay should be "Mai" right ?

correct.

**Comment by kunj-sangani** [comment]
Hi @aaclage ,

Update as mentioned can you please verify again
Sorry it took a long time :)

File: src/loc/pt-br.ts [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+    ListItemAttachmentsfileDeleteError: 'Erro ao excluir arquivo: {0}, motivo {1}',
+    ListItemAttachmentserrorLoadAttachments: 'Erro ao carregar o anexo do item da lista, motivo: {0}',
+    ListItemAttachmentsconfirmDelete: 'Tem certeza de que deseja enviar o anexo {0} para a lixeira do site?',
+    ListItemAttachmentsdialogTitle: 'Anexo de item de lista',
+    ListItemAttachmentsdialogOKbuttonLabel: 'OK',
+    ListItemAttachmentsdialogCancelButtonLabel: 'Cancelar',
+    ListItemAttachmentsdialogOKbuttonLabelOnDelete: 'Excluir',
+    ListItemAttachmentsuploadAttachmentDialogTitle: 'Juntar anexo',
+    ListItemAttachmentsuploadAttachmentButtonLabel: 'Juntar anexo',
+    ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não anexado, motivo: {1}',
```
Comment (aaclage) [inline src/loc/pt-br.ts:70]: ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não anexado, motivo: {1}', should be 
 ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não **foi** anexado, motivo: {1}',

**Comment by joelfmrodrigues** [comment]
@kunj-sangani many thanks for the PR 
@aaclage thanks for reviewing it 

There are some little things where I am not 100% sure if they are correct for pt-br, but will merge as is and will try to get a colleague from Brasil to double check :)

**Comment by joelfmrodrigues** [comment]
@kunj-sangani forgot to mention, but feel free to submit a PR also to dev branch if you get the chance.

**Pull request merged**